pkg/cloud/cloud.go

```diff
@@ -299,6 +299,8 @@ func (c *cloud) CreateDisk(ctx context.Context, volumeName string, diskOptions *
 		throughput = int64(diskOptions.Throughput)
 	case "":
```

Comment:
This can be merged with `case VolumeTypeGP3`

Reply:
Here we have `DefaultVolumeType` for "" (instead of `gp3`), and we can change the default volume type at L114. I don't think it can be merged to `case VolumeTypeGP3`. And the `default:` here handles all the invalid inputs.